package.json

```diff
@@ -35,20 +42,15 @@
   "main": "dist/index.js",
   "name": "react-amplitude-hooks",
   "peerDependencies": {
-    "react": "^16.8.0 || ^17"
+    "react": "^16.8.0 || ^17 || ^18"
```

Comment:
Should include react 19 -- also I think we can remove 16 as a supported version.